src/js/components/FormField/FormField.js

```diff
@@ -234,6 +234,11 @@ const FormField = forwardRef(
     } else if (normalizedError) {
       borderColor =
         (themeBorder && themeBorder.error.color) || 'status-critical';
+    } else if (focus) {
```

Comment:
This comment goes beyond just this change.

If the theme does have `formField.focus` but does not set `border` in it, we won't be using the default border color, line 243 in the new code. I would think I could set the border color in the theme for the default case and have that work for disabled, error, and focus cases if I don't wan't the border to change for them. Maybe I'm changing the background instead as an indicator? Or, maybe focus is reflected only via shadow, which is already taken care of.

Reply:
This is a good point if formFieldTheme.focus was defined but the color. was not defined your correct it would not set border color to. line 243.. Instead Im wondering if within border color for focus we can have 

```

      borderColor =
        (formFieldTheme.focus &&
        formFieldTheme.focus.border &&
        formFieldTheme.focus.border.color) || (themeBorder && themeBorder.color) || 'border';

```

This would be a little similar to border color when there is an error the fall back is `status-critical` 
and then doing something similar with `disabled-borderColor`

Reply:
You're on the right track, we just need to restructure the logic appropriately. My inclination is to do something like:
```
let borderColor;
if (focus && formFieldTheme.focus && formFieldTheme.focus.border.color && formFieldTheme.focus.border.color) borderColor = formFieldTheme.focus.border.color;
else if (...)
```

Reply:
When I get to publishing my Button pull request, I've got some patterns that might apply as well here. But, not quite ready yet.